README.rst

```diff
@@ -3,3 +3,35 @@ Python Infrastructure
 
 * `Documentation <http://infra.psf.io/>`_
 * `IRC: #python-infra on Freenode <http://webchat.freenode.net?channels=%23python-infra>`_
+
+
+Development:
+------------
```

Comment:
The Kallithea documentation should go in the docs instead of the readme. Probably this can just be moved to ``docs/services/kallithea.rst``.